.pnpmfile.cjs

```diff
@@ -0,0 +1,66 @@
+//@ts-check
+
+// Based on https://gist.github.com/dvins/33b8fb52480149d37cdeb98890244c5b
```

Comment:
Nit: also adding a brief overview of what this module does / why it exists above this would be helpful